scripts/etch-language-tests/uint256.etch

```diff
@@ -2,6 +2,7 @@ function main()
 
    var uint256 = UInt256(100u64);
 
-   assert(toString(uint256) == "6400000000000000000000000000000000000000000000000000000000000000");
+   var expected_hex_str_in_big_endian_repr = "0000000000000000000000000000000000000000000000000000000000000064";
```

Comment:
that's actually little endian representation

Reply:
That should be big endian - the least significant byte is on the most right side of the array.
For example *big* endian representation of the `uint64(100<DEC>)` in memory will be `{0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x64}`, and *little* endian will be `{0x64, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00}`

Reply:
yes, you're right, I was confused, ignore this comment :)